Resources/Prototypes/_RMC14/Roles/Ranks/UNCM/Other/senior_enlisted_advisor.yml

```diff
@@ -0,0 +1,23 @@
+- type: rank
+  parent: RMCBaseRankSergeantMajor
+  id: RMCRankSEASergeantMajor
+  requirements:
+  - !type:RoleTimeRequirement
+    role: CMJobSeniorEnlistedAdvisor
+    time: 630000 # 175 hours
+
+- type: rank
+  parent: RMCBaseRankFirstSergeant
+  id: RMCRankSEAFirstSergeant
+  requirements:
+  - !type:RoleTimeRequirement
+    role: CMJobSeniorEnlistedAdvisor
+    time: 252000 # 70 hours
+
+- type: rank
+  parent: RMCBaseRankMasterSergeant
+  id: RMCRankSEAMasterSergeant
+  requirements:
+  - !type:RoleTimeRequirement
+    role: CMJobSeniorEnlistedAdvisor
+    time: 90000 # 25 hours
```

Comment:
SEA does get Master Gunnery Sergeant after First Sergeant. The actual playtime involved is not in the code, but neither are any of the ranks above Gunnery Sergeant

Reply:
added this